### PR TITLE
Add script to delete weeded holdings for Li

### PR DESCRIPTION
**Pull request by mxtthias**

### CODE REVIEW

**Comment by mxtthias** [comment]
Hold off on this one, seems like we'll do it another way.

**Comment by mxtthias** [comment]
Updated to use a list of supplied IDs instead. Open for review!

File: whelktool/scripts/cleanups/2019/02/delete-weeded-holdings-for-Li/delete-weeded-holdings-for-Li.groovy [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
+ * This deletes all holdings for Li with availability 'UTGALLRAD'.
+ *
+ * See LXL-2275 for more info.
+ *
+ */
+String SIGEL_TO_DELETE = 'Li'
+String BIB_ID_FILE = 'lxl-2275-ids.txt'
```
Comment (sarwik) [inline whelktool/scripts/cleanups/2019/02/delete-weeded-holdings-for-Li/delete-weeded-holdings-for-Li.groovy]: Where can I find the file with the id's? Missed to add to branch?

Comment (mxtthias) [reply whelktool/scripts/cleanups/2019/02/delete-weeded-holdings-for-Li/delete-weeded-holdings-for-Li.groovy]: Doh! Fixed now! :)

**Pull request merged**